### PR TITLE
Translate tasks/2d_body_keypoint.md

### PR DESCRIPTION
**Pull request by ZhiYuan-REN**
This is my first try!
I change a very little thing in /mmpose/docs_zh-CN/tasks/2d_body_keypoint.md.
This PR is to help me familar with the whole PR process.

### CODE REVIEW

**Comment by jin-s13** [comment]
Thanks. @ZhiYuan-REN  Would you like to help translate this part? 
Please sign CLA.

**Comment by jin-s13** [comment]
Please follow https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md to setup pre-commit.

File: configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/mpii/hrnet_w32_mpii_256x256.py [hunk 1]
```diff
@@ -128,7 +128,7 @@
 
 data_root = 'data/mpii'
 data = dict(
-    samples_per_gpu=64,
+    samples_per_gpu=8,
```
Comment (jin-s13) [inline configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/mpii/hrnet_w32_mpii_256x256.py:131]: This change is not related to this PR.

File: docs_zh-CN/tasks/2d_body_keypoint.md [hunk 2]
```diff
@@ -1,3 +1,500 @@
...
+  booktitle={European conference on computer vision},
+  pages={740--755},
+  year={2014},
+  organization={Springer}
+}
+```
+
+</details>
+
+对 [COCO](http://cocodataset.org/) 数据集来说， 请从此链接 [COCO download](http://cocodataset.org/#download) 下载， 2017 Train/Val 对于 COCO 关键点的训练和评估是非常必要的。
```
Comment (innerlee) [inline docs_zh-CN/tasks/2d_body_keypoint.md:40]: `来说， 请从此`

extra space before `请`

File: docs_zh-CN/tasks/2d_body_keypoint.md [hunk 3]
```diff
@@ -1,3 +1,500 @@
...
+[HRNet-Human-Pose-Estimation](https://github.com/HRNet/HRNet-Human-Pose-Estimation) 提供了 COCO val2017 的检测结果，来帮助重现我们的多人姿态估计的结果。
+请从 [OneDrive](https://1drv.ms/f/s!AhIXJn_J-blWzzDXoz5BeFl8sWM-) 或 [GoogleDrive](https://drive.google.com/drive/folders/1fRUDNUDxe9fjqcRZ2bnF_TKMlO0nB_dk?usp=sharing)下载。
+在 $MMPOSE/data 目录下下载并提取， 使得文件呈如下的格式：
+可选地, 为了在 COCO'2017 上评估， 请下载 [image-info](https://download.openmmlab.com/mmpose/datasets/person_keypoints_test-dev-2017.json)。
```
Comment (innerlee) [inline docs_zh-CN/tasks/2d_body_keypoint.md:44]: 为了在 COCO'2017 test-dev 上评估

Comment (ZhiYuan-REN) [reply docs_zh-CN/tasks/2d_body_keypoint.md:44]: 收到

File: docs_zh-CN/tasks/2d_body_keypoint.md (hunk 2 above)
Comment (ZhiYuan-REN) [reply docs_zh-CN/tasks/2d_body_keypoint.md:40]: 收到

**Comment by jin-s13** [comment]
subdir `poseval` should not be uploaded.

**Comment by ZhiYuan-REN** [comment]
what a shame! I can not remove poseval. I will relaunch this PR!
